.release/ci.hcl

```diff
@@ -0,0 +1,110 @@
+schema = "1"
+
+project "consul-terraform-sync" {
+  team = "consul api tooling"
+  slack {
+    notification_channel = "#feed-releng" #TODO update slack channel
```

Comment:
Was wondering if you recommend us switching back to our notification channel after this PR? Want to make sure if you have any plans monitoring this in #feed-releng

Reply:
Hi @lornasong! Right now, our tooling will only send notifications to #feed-releng but we will be working very soon on getting this enabled for individual team channels and will then circle back and update this value in the CTS OSS/ENT repos 👍

Reply:
Hello @sarahethompson! That sounds good! Thanks for clarifying :)